pkgs/tools/audio/patray/default.nix

```diff
@@ -29,11 +29,9 @@ python3.pkgs.buildPythonApplication rec {
 
   nativeBuildInputs = [ qt5.wrapQtAppsHook ];
   postFixup = ''
-    wrapQtApp $out/bin/patray
+    wrapQtApp $out/bin/patray --prefix QT_PLUGIN_PATH : ${qt5.qtbase}/${qt5.qtbase.qtPluginPrefix}
```

Comment:
Instead of manually doing `wrapQtAppsHook`'s job here, you just need to add at least
```nix
buildInputs = [
  qt5.qtbase
];
```
so `wrapQtApp` can actually pick up the dependency and set the variable.

Reply:
I tried that but it didn't work.